### PR TITLE
[DEV-195] fix php documentation

### PR DESCRIPTION
**Pull request by joelmendez1**

### CODE REVIEW

File: server/controllers/article/edit.php [hunk 1]
```diff
@@ -24,7 +24,6 @@
  *
  * @apiUse NO_PERMISSION
  * @apiUse INVALID_TOPIC
- * @apiUse INVALID_FILE
```
Comment (ivandiazwm) [inline server/controllers/article/edit.php:27]: I think this exists in `Controller.php` in `uploadImages`

File: server/controllers/ticket/add-custom-response.php [hunk 2]
```diff
@@ -19,7 +19,6 @@
  * @apiParam {String} language Language of the response.
  *
  * @apiUse NO_PERMISSION
- * @apiUse INVALID_NAME
```
Comment (ivandiazwm) [inline server/controllers/ticket/add-custom-response.php:22]: `INVALID_TITLE` is missing here

File: server/controllers/ticket/comment.php [hunk 3]
```diff
@@ -25,8 +25,6 @@
  * @apiUse NO_PERMISSION
  * @apiUse INVALID_CONTENT
  * @apiUse INVALID_TICKET
- * @apiUse INVALID_TOKEN
- * @apiUse INVALID_FILE
```
Comment (ivandiazwm) [inline server/controllers/ticket/comment.php:29]: same here for INVALID_FILE

File: server/controllers/ticket/create.php [hunk 4]
```diff
@@ -32,7 +32,6 @@
  * @apiUse INVALID_LANGUAGE
  * @apiUse INVALID_CAPTCHA
  * @apiUse INVALID_EMAIL
- * @apiUse INVALID_FILE
```
Comment (ivandiazwm) [inline server/controllers/ticket/create.php:35]: here too

**Review by ivandiazwm — CHANGES_REQUESTED** [review]

**Comment by mredigonda** [comment]
Are there any updates on this PR?

If you have any please upload them and re-request review from Ivan 👍🏼.

File: server/controllers/system/init-settings.php [hunk 5]
```diff
@@ -19,15 +19,13 @@
...
  * @apiParam {String} server-email Email from where automated emails will be sent.
  * @apiParam {String} smtp-host SMTP Server address.
- * @apiParam {String} smtp-port SMTP Server port.
  * @apiParam {String} smtp-user SMTP Authentication User.
  * @apiParam {String} smtp-pass SMTP Authentication Password.
  * @apiParam {String} allow-attachments Indicates if files can be attached to tickets and comments.
  * @apiParam {String} title Title of the support center
  * @apiParam {String} url Url of the frontend client.
  * @apiParam {Boolean} mandatory-login Indicates if the login is mandatory.
  * @apiParam {Number} default-department-id Indicates the id of the default department
```
Comment (mredigonda) [inline server/controllers/system/init-settings.php:28]: This also isn't being used.

File: server/controllers/system/stats.php [hunk 6]
```diff
@@ -20,7 +20,6 @@
  * @apiParam {Number[]} owners The ids of the owners for the custom stats.
  * 
  * @apiUse NO_PERMISSION
- * @apiUse INVALID_PERIOD
```
Comment (mredigonda) [inline server/controllers/system/stats.php:23]: Remove it from `ERRORS.php` too.

File: server/controllers/ticket/comment.php (hunk 3 above)
Comment (mredigonda) [reply server/controllers/ticket/comment.php:29]: I guess it's used implicitly by the `FileUploader`? In that case (if this path can return that error) then let's leave it.

File: server/controllers/ticket/edit-title.php [hunk 7]
```diff
@@ -19,7 +19,6 @@
  *
  * @apiUse NO_PERMISSION
  * @apiUse INVALID_TITLE
- * @apiUse INVALID_TOKEN
```
Comment (mredigonda) [inline server/controllers/ticket/edit-title.php:22]: Seems we are also using `INVALID_TICKET` error here.

**Review by mredigonda — COMMENTED** [review]
I noticed some small details to complete with this PR, after that, we can merge!

**Review by mredigonda — APPROVED** [review]
LGTM 🎉

**Pull request merged**